### PR TITLE
Fix 10565 - Extract freetype to its own baseline

### PR DESCRIPTION
**Pull request by estebanlm**

### CODE REVIEW

**Comment by estebanlm** [comment]
error not related

File: src/BaselineOfFreeType/BaselineOfFreeType.class.st [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+{ #category : #baselines }
+BaselineOfFreeType >> baseline: spec [
+	<baseline>
+	
+	spec for: #common do: [ 
+		spec package: 'FreeType'.
+		spec package: 'FreeType-Morphic'.
+		spec package: 'FreeType-Tests'.
```
Comment (astares) [inline src/BaselineOfFreeType/BaselineOfFreeType.class.st:14]: Dependencies among packages should be declared, no?

Like for example:
```Smalltalk
spec package: 'FreeType-Tests' with: [ spec requires: #('FreeType' 'FreeType-Morphic') ]
```

**Comment by estebanlm** [comment]
not really needed, since dependency is explicit in groups

**Comment by astares** [comment]
@estebanlm 

It's also possible to load (single) packages - or reference them when loading from other baselines, not only the groups. So dependencies should be defined clearly on package level, additonally for the groups  IMHO

**Comment by MarcusDenker** [comment]
random failure:

- [CI] Failing randomly: StPlaygroundPageTest>>testContents #10519

File: src/BaselineOfFreeType/BaselineOfFreeType.class.st (hunk 1 above)
Comment (MarcusDenker) [reply src/BaselineOfFreeType/BaselineOfFreeType.class.st:14]: I will merge the current state for now...

**Review by MarcusDenker — APPROVED** [review]

**Pull request merged**